packages/alfa-comparable/src/comparable.ts

```diff
@@ -1,9 +1,15 @@
+export enum Comparison {
+  Smaller = -1,
```

Comment:
```suggestion
  Less = -1,
```

Great thinking introducing an enum for this though, I dig it! 💯